mtenn/_version.py

```diff
@@ -1 +1 @@
-__version__ = "0.1.0"
+__version__ = "0.2.0"
```

Comment:
If you do this we should make tags for 0.2.0 right after merge.

Reply:
I can do that once this is in.

Reply:
I'm new to all this versioning stuff, so if there's a different/better way to do this I'm happy to do that instead!